cmd/kops/get.go

```diff
@@ -39,12 +45,45 @@ var getCmd = GetCmd{
 const (
 	OutputYaml  = "yaml"
 	OutputTable = "table"
+	OutputJSON  = "json"
 )
 
 func init() {
 	cmd := getCmd.cobraCommand
 
 	rootCommand.AddCommand(cmd)
 
-	cmd.PersistentFlags().StringVarP(&getCmd.output, "output", "o", OutputTable, "output format.  One of: table, yaml")
+	cmd.PersistentFlags().StringVarP(&getCmd.output, "output", "o", OutputTable, "output format.  One of: table, yaml, json")
+}
+
+type marshalFunc func(obj runtime.Object) ([]byte, error)
+
+func marshalToStdout(obj runtime.Object, marshal marshalFunc) error {
+	b, err := marshal(obj)
+	if err != nil {
+		return err
+	}
+	_, err = os.Stdout.Write(b)
+	if err != nil {
+		return fmt.Errorf("error writing to stdout: %v", err)
+	}
+	return nil
+}
+
+// v must be a pointer to a marshalable object
+func marshalYaml(obj runtime.Object) ([]byte, error) {
+	y, err := api.ToVersionedYaml(obj)
+	if err != nil {
+		return nil, fmt.Errorf("error marshaling yaml: %v", err)
+	}
+	return y, nil
+}
+
+// v must be a pointer to a marshalable object
+func marshalJSON(obj runtime.Object) ([]byte, error) {
+	j, err := json.MarshalIndent(obj, "", "  ")
```

Comment:
We probably should build api.ToVersionedJson, but I'm also still learning the API machinery so we can leave this for now :-)